public/Notes from Gavn on Git

```diff
@@ -0,0 +1,36 @@
+1) stage everything for a commit 

```

Comment:
I don't think we should commit this file to our package. You should keep a copy of it on your computer though...
Copy it over to somwhere else (you could create your own package on git for notes and use it for practice etc).  Delete it from the directory then `git add .` -> `git commit` ->  `git push` it will update this pull request automatically.